### PR TITLE
Feature 1351 Config auslagern

### LINKED ISSUE #1351: ION.defaultAccessToken in Config auslagern
**Beschreibung**
Der AccessToken, mit dem Daten von Cesium-ION  in die Viewer-Applikation integriert werden, ist im Code hard-codiert. Der AccessToken soll in ein Config-File (z.B. env) ausgelagert werden.

Der Token ist an folgenden Stellen im Code hinterlegt:
[Cesium-ION AccesTokens ](https://github.com/search?q=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI0YjNhNmQ4My01OTdlLTRjNmQtYTllYS1lMjM0NmYxZTU5ZmUiLCJpZCI6MTg3NTIsInNjb3BlcyI6WyJhc2wiLCJhc3IiLCJhc3ciLCJnYyJdLCJpYXQiOjE1NzQ0MTAwNzV9.Cj3sxjA_x--bN6VATcN4KE9jBJNMftlzPuA8hawuZkY&type=code)

Der Token besitzt folgende Rechte: 
Access
This token provides access to the following scopes:
- assets:list
- assets:read
- assets:write
- geocode

**ToDos**
- [x] Prüfen, ob der Token Schreibberechtigungen benötigt. wenn nicht kann das entsprechende Recht dem Token entzogen werden.
- [ ] AccessToken in ein Config-File auslagern. Es gibt eine Datei in der Konstanten gespeichert werden ([ui/src/constants.ts](https://github.com/swisstopo/swissgeol-viewer-app/blob/master/ui/src/constants.ts)), vielleicht könnten wir diese Daten auch für den cesium ion AccessToken nutzen?

**Acceptance Criteria**
- [ ] AccessToken ist in ein Config-File auslagert und die Daten von Cesium-ION werden korrekt angezeigt

### PR DESCRIPTION
**Pull request by vej-ananas**
Resolves #1351

### CODE REVIEW

File: ui/src/layers/helpers.ts [hunk 1]
```diff
@@ -1,259 +1,260 @@
...
+  primitive.customShader = getVoxelShader(config);
+  primitive.jitter = false;
+
+  return primitive;
+}
+export async function create3DTilesetFromConfig(viewer: Viewer, config: LayerConfig, tileLoadCallback) {
+  let resource: string | IonResource; // | AmazonS3Resource
+  if (config.aws_s3_bucket && config.aws_s3_key) {
+    throw new Error('Amazon S3 resources is not implemented');
+    // resource = new AmazonS3Resource({
```
Comment (vej-ananas) [inline ui/src/layers/helpers.ts:95]: I am still trying to find out if this is used

File: api/.env [hunk 2]
```diff
@@ -29,4 +30,8 @@ S3_ENDPOINT=http://minio:9000
 # Cognito
 COGNITO_AWS_REGION=eu-west-1
 COGNITO_CLIENT_ID=10h1tga4i933buv25lelalmtrn
+COGNITO_IDENTITY_POOL_ID=eu-west-1:aa0d145d-228e-40be-bb73-a9a2c83879df
 COGNITO_POOL_ID=eu-west-1_dbfEb2FuH
+
+# ION
+ION_DEFAULT_ACCESS_TOKEN='eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI0YjNhNmQ4My01OTdlLTRjNmQtYTllYS1lMjM0NmYxZTU5ZmUiLCJpZCI6MTg3NTIsInNjb3BlcyI6WyJhc2wiLCJhc3IiLCJhc3ciLCJnYyJdLCJpYXQiOjE1NzQ0MTAwNzV9.Cj3sxjA_x--bN6VATcN4KE9jBJNMftlzPuA8hawuZkY'
```
Comment (daniel-va) [inline api/.env:37]: Is it okay to have this in the GitHub repo?

File: ui/src/api/client-config.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export interface ClientConfig {
+    'env': 'dev' | 'int' | 'prod',
+    'ion_default_access_token': string,
+    'auth': {
+        'cognito_client_id': string,
+        'cognito_pool_id': string,
+        'cognito_identity_pool_id': string,
+        'cognito_aws_region': string,
+    }
```
Comment (daniel-va) [inline ui/src/api/client-config.ts:9]: Is there a reason for quoting the keys here?

File: ui/src/api/config.service.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  constructor() {
+    this.apiUrl = API_BY_PAGE_HOST[window.location.host];
+  }
+
+  async getConfig(): Promise<ClientConfig | null> {
+    const headers = {};
+    try {
+      const response = await fetch(`${this.apiUrl}/client-config`, {
+        method: 'GET',
+        headers: headers,
```
Comment (daniel-va) [inline ui/src/api/config.service.ts:16]: Are we expecting any headers in the future? If not, this field & the `headers` variable could just be removed.

**Review by daniel-va — APPROVED** [review]

**Comment by daniel-va** [comment]
Do we report non-functional changes in the CHANGELOG? If yes, then we might as well add this feature.

**Comment by daniel-va** [comment]
# Squash commits before merge

File: api/.env (hunk 2 above)
Comment (vej-ananas) [reply api/.env:37]: Yes, the same token as before - it is read only. maybe we remove it in the future.

**Review by daniel-va — APPROVED** [review]

**Pull request merged**